src/Components/Article.jsx

```diff
@@ -0,0 +1,35 @@
+import { useState, useEffect } from "react"
+import { getArticle } from "../../api"
+import { useParams } from "react-router-dom"
+import Button from '@mui/material/Button';
+import formatDate from "./FormatDate";
+
+const Article = () => {
+    const [article, setArticle] = useState({});
+    const [isLoading, setIsLoading] = useState(true);
+    const { article_id } = useParams();
+
+    useEffect(() => {
+        getArticle(article_id).then((article) => {
+            setArticle(article)
+            setIsLoading(false)
+        })
+    },[])
+  return isLoading ? (
+    <h2>Loading...</h2>
+  ) : (
```

Comment:
This is really nice loading state, I would recommend having `article_id` as a dependency in the array for the _useEffect_ so that if you want people to be able to navigate directly from one article to another it will re-fetch the new article